books/free-programming-books-de.md

```diff
@@ -55,7 +55,7 @@
 
 ### Assembly Language
 
-* [PC Assembly Language](http://drpaulcarter.com/pcasm) - Paul A. Carter
```

Comment:
link works

Reply:
Hi @eshellman  Thanks for the review. However there are two points I should mention
- This is not the pdf direct link , its more like an article on it
http://drpaulcarter.com/pcasm
![PCASM - Google Chrome 04_10_2022 09_53_45](https://user-images.githubusercontent.com/62697124/193744842-6e0424f9-a054-44dd-b4ca-0294530f267d.png)

as compared to 
https://pacman128.github.io/static/pcasm-book.pdf
![PCASM - Google Chrome 04_10_2022 09_53_50](https://user-images.githubusercontent.com/62697124/193744947-3bf56bf8-9a43-473a-b6a2-cf24222e7059.png)

- Also there is also the fix for this link  in this commit
[L184] 404 https://symfony.com/doc/5.0/the-fast-track/de/index.html